source/guides/scale-mattermost.rst

```diff
@@ -17,7 +17,9 @@ Scale Mattermost
     Scale up to 88000 users </scale/scale-to-88000-users>
     High availability cluster </scale/high-availability-cluster>
     Elasticsearch </scale/elasticsearch>
-    Performance monitoring </scale/performance-monitoring>
+    Monitor performance using the Metrics plugin </scale/metrics-plugin>
+    Monitor performance using dashboards </scale/performance-monitoring>
```

Comment:
Wondering if these two are sounding like alternative to each other. What about naming them like: 
```suggestion
    Collect performance metrics using the Metrics plugin </scale/metrics-plugin>
    Deploy Prometheus and Grafana for performance monitoring </scale/performance-monitoring>
```